net.pdl2ork.PdL2Ork.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.pdl2ork.PdL2Ork</id>
```

Comment:
`pdl2ork.net` doesn't seem to exist. This must be a domain under the control of the ustream developers.

Reply:
I see. I was simply going by what another fork had on flathub. My URL is l2ork.music.vt.edu. Should I do edu.vt.music.PdL2Ork?

Reply:
So you didn't even read the questions you aknowledged:

> I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).

Nor the documentation.

Two wrongs don't make a right.

Reply:
> So you didn't even read the questions you aknowledged:
> 
> > I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
> 
> Nor the documentation.
> 
> Two wrongs don't make a right.

As a faculty at a university who owns this domain, I don't own the URL, but I do manage the server and am in charge of managing that specific subdomain. Should I register a separate, external domain and create a site (e.g. via github.io) just to be able to have a flatpak? If so, can I set it up as a redirect to our current site or github?

Reply:
you can simply use something associated to the github repository (`io.github.`) It's the low friction way.

Reply:
I think I've addressed all other issues and have pushed changes to the branch. I will look into creating a site github.io tomorrow. Then I only have to add one more file that I missed in the build process and we should be hopefully good to go, pending your approval. Thanks for your help.

Reply:
So, I just created a site that is accessible at https://pd-l2ork.github.io/pd-l2ork/

Should therefore my ID be io.github.Pd-L2Ork, io.github.pdl2ork.Pd-L2Ork, io.github.pd-l2ork.Pd-L2Ork (last one has a dash in penultimate and last namespace, and the convention, unless I am misreading it, only allows dashes in the last one)? Thanks.

Reply:
> io.github.pdl2ork.Pd-L2Ork

This is fine, you can replace the last dash with underscore https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic if you want

Reply:
@bbhtt thank you. I will make this change shortly and upload the updated version asap.

Reply:
> I will look into creating a site github.io tomorrow.

this is not even necessary.

Reply:
> This is fine, you can replace the last dash with underscore

not can. should.

Reply:
Can I omit the underscore and just do io.github.pdl2ork.PdL2Ork?

Reply:
no. it SHOULD match `io.github.pd_l2ork.*`

Reply:
OK, thanks. Will retitle them...